htmlmin/command.py

```diff
@@ -152,8 +150,7 @@ def main():
   if args.input_file:
     inp = codecs.open(args.input_file, encoding=args.encoding)
   else:
-    inp = codecs.getreader(
-      sys.stdin.encoding or locale.getpreferredencoding())(sys.stdin)
+    inp = io.open(0, encoding=args.encoding)
```

Comment:
Let's just update the help text:

Encoding to read and write with. Default 'utf-8'. When reading from stdin, attempts to use the system's encoding before defaulting ot utf-8.\n\n